helm/spark-yotpo/templates/sparkapplication.yaml

```diff
@@ -138,18 +138,18 @@ spec:
     {{ toYaml .Values.tolerations | indent 6 }}
     {{- end }}
     {{- if .Values.jmxExporterJar }}
-monitoring:
-  exposeDriverMetrics: true
-  exposeExecutorMetrics: true
-  prometheus:
-    port: {{ .Values.jmxPort | default 8090 }}
-    jmxExporterJar: {{ .Values.jmxExporterJar }}
-  {{- end }}
-  {{- if .Values.volumes }}
-volumes:
-  {{ toYaml .Values.volumes | indent 3 }}
-  {{- end }}
-  {{- if .Values.nodeSelector }}
-nodeSelector:
-  {{ toYaml .Values.nodeSelector | indent 4 }}
-  {{- end }}
\ No newline at end of file
+  monitoring:
+    exposeDriverMetrics: true
+    exposeExecutorMetrics: true
+    prometheus:
+      port: {{ .Values.jmxPort | default 8090 }}
+      jmxExporterJar: {{ .Values.jmxExporterJar }}
+    {{- end }}
+    {{- if .Values.volumes }}
+  volumes:
```

Comment:
@giladw are u sure volumes suppose to be idented like this?

Reply:
yes i think everything is under spec.
here in an example with volumes
https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/de0bdf1275e77b6d053d408c045c427aa94a42ab/examples/spark-pi.yaml

Reply:
btw, this is the CRD :
https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/5b2fe9218108baa6456eb3472196d5eb55b41efd/manifest/crds/sparkoperator.k8s.io_sparkapplications.yaml